src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,120 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import exception.InvalidDeadlineException;
+import exception.InvalidEventException;
+import exception.InvalidTodoException;
+import ui.Ui;
+
+public class Storage {
+
+    /**
+     * Write the content of the list into a file
+     * @param list The task list
+     */
+    public void writeFile(ArrayList<Task> list) {
+        try {
+            StringBuilder replacementText = new StringBuilder();
+            createFile("data/duke.txt");
+            for (int i = 0; i < list.size(); i++) {
+                if (list.get(i) instanceof Todo) {
+                    replacementText.append(list.get(i).toString() + "\n");
+                } else {
+                    replacementText.append(list.get(i).toString() + " on " + list.get(i).getDate() + "\n");
+                }
+            }
+            appendToFile("data/duke.txt", replacementText.toString());
+        } catch (IOException e) {
+            System.out.println("Folder data not found! " + e);
+        }
+    }
+
+    /**
+     * Adds a string to a file
+     * @param filePath The file path (destination)
+     * @param textToAppend The content of the file
+     */

```

Comment:
Should there be an empty line between the description and the parameter section? This styling seems to only be in certain classes and not others.

Reply:
Sorry for the confusion. Will make it more consistent.